sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/TimeSeriesInsightsClient.cs

```diff
@@ -94,40 +96,101 @@ protected TimeSeriesInsightsClient()
         }
 
         /// <summary>
-        /// Returns the model settings which includes model display name, Time Series ID properties and default type ID. Every Gen2 environment has a model that is automatically created.
+        /// Gets the scope for authentication/authorization policy.
         /// </summary>
-        /// <param name="clientSessionId"> Optional client session ID. Service records this value. Allows the service to trace a group of related operations across services, and allows the customer to contact support regarding a particular group of requests. </param>
-        /// <param name="cancellationToken"> The cancellation token to use. </param>
-        /// <returns>The deserialized application/json model settings digital twin and the http response <see cref="Response{ModelSettingsResponse}"/>.</returns>
-        /// <example>
-        /// This sample demonstrates getting and deserializing a model settings.
-        ///
-        /// <code snippet="Snippet:TimeSeriesInsightsGetModelSettings">
-        /// // Get the model settings for the time series insights environment
-        /// Response&lt;Models.ModelSettingsResponse&gt; response = await client.GetAsync().ConfigureAwait(false);
-        /// Console.WriteLine($&quot;Retrieved model {response.Value.ModelSettings.Name}.&quot;);
-        /// </code>
-        /// </example>
-        public virtual Task<Response<ModelSettingsResponse>> GetAsync(string clientSessionId = null, CancellationToken cancellationToken = default)
+        /// <returns>List of scopes for the specified endpoint.</returns>
+        internal static string[] GetAuthorizationScopes() => s_tsiDefaultScopes;
+
+        /// <summary>
+        /// Gets model settings asynchronously.
```

Comment:
Maybe reword to "Gets Time Series Insights model settings asynchronously.". I need to check with Eric on whether Time Series should be upper case or we can lower case it.